modules/document_repository/test/TestPlan

```diff
@@ -1,44 +1,46 @@
-1. User has access to document repository iff he/she has permission “Access all documents” or is the super user.
+1. User has access to document repository if they have the "View and upload files in Document Repository” permission or is the super user.
    [Automation Testing]
 2. Create a category and a sub category.
    [Automation Testing]
 3. Check that the comments for a category are displayed properly. Also check that they can be edited/added.
    [Automation Testing]
-3. Upload/delete a file.
+4. Upload a file.
    [Manual Testing]
-4. Edit document property and save. Check that changes were saved properly.
+5. User is able to delete a file if they have the "Delete files in Document Repository" permission or is the super user.
    [Manual Testing]
-5. Download a file and check content.
+6. Edit document property and save. Check that changes were saved properly.
    [Manual Testing]
-6. Add files in repository to test search according to file name, file type, version, site and uploaded by.
+7. Download a file and check content.
    [Manual Testing]
-7. Check that expand/collapse works for non-empty categories. Check that when performing a search, the categories
+8. Add files in repository to test search according to file name, file type, version, site and uploaded by.
+   [Manual Testing]
+9. Check that expand/collapse works for non-empty categories. Check that when performing a search, the categories
    containing the resulting files (and only those) are shown in a list format
    [Manual Testing]
-8. Check that if a category contains special characters (e.g. space, dot or comma) it can be expanded.
+10. Check that if a category contains special characters (e.g. space, dot or comma) it can be expanded.
    [Manual Testing]
-9. Add two files with the same name but with different contents in two different categories. Verify that, when 
+11. Add two files with the same name but with different contents in two different categories. Verify that, when 
    downloaded, their contents are different.
    [Manual Testing]
-10. Edit a file in the repository: check that “Last modified on” date is updated.
+12. Edit a file in the repository: check that “Last modified on” date is updated.
     [Manual Testing]
-11. Upload a file using another user and then update it. Check that “Last updated by” field is updated.
+13. Upload a file using another user and then update it. Check that “Last updated by” field is updated.
```

Comment:
This column doesn't actually exist. Should this remain in the Testplan? Or should a PR be created to address this?